app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Notification.kt

```diff
@@ -60,7 +60,7 @@ import com.hedvig.android.design.system.internals.InternalSnackBar
 import com.hedvig.android.design.system.internals.NotificationColors
 
 @Composable
-fun HedvigNotificationCard(
+fun HedvigInfoCard(
```

Comment:
I named it that since the Figma tab is named that, and I figured it'd then be hard to look at figma, see it named one way, and then in the code we have it named some other way.
Of course "notification" feels completely unrelated in our code, so I can definitely see why you renamed it, but I wonder if we should also ask design to consider renaming the component there too?